.snyk

```diff
@@ -0,0 +1,125 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JAVA-ORGAPACHETOMCAT-460328:
+    - '*':
+        reason: Imported from vulnerability scanner suppression list
+        expires: 2026-10-13T00:00:00.000Z
```

Comment:
❓ Why did you choose this expiry date?

Reply:
I chose this date because it will be the EOL of 7.23.0, the current SNAPSHOT.
Now, thinking about it again, maybe it's not good to have it expiry because the 7.24 branch will probably inherit this file, and the ignore rules would expire before the EOL there. Unfortunately, it's [not possible to have no expiry date](https://docs.snyk.io/snyk-cli/commands/ignore#expiry-less-than-expiry-greater-than). The default is 30 days.

Reply:
Do you have a better idea for an expiry date?

Reply:
Can we set it to `2032-05-01T00:00:00.000Z`? That would roughly cover EOL of 7.24.0.

Reply:
Sure, I can set that for all ignores.

Reply:
Done; all ignores expire in 2032 now.